### ISSUE TITLE
Reorganize BRSI collection

### ISSUE THREAD

**Issue by hoodriverheather** [EDRN/edrn-skin#1]
The retrospective Images for Duke line up with the blood in the BRS reference set which is being used for validation, so those images should be put in a separate LabCAS instance.  Moffitt’s retrospective images don’t have blood and we should ask what their purpose is.  If it is just training, then maybe they can be kept with the training set, not certain.  So yes, please ask.  Thanks - Jackie
 
From: Kincaid, Heather L (US 398G-Affiliate) <heather.kincaid@jpl.nasa.gov> 
Sent: Thursday, February 10, 2022 7:31 AM
To: Stelling, Deanna L <dstellin@fredhutch.org>; Dahlgren, Jackie <jdahlgre@fredhutch.org>
Cc: Zhao, Yingqi <yqzhao@fredhutch.org>
Subject: Re: [EXTERNAL] RE: BRSI Imaging Set
 
Hi Deanna and Jackie,
Thanks. Let us know how you want handle the Retrospective participants and we will make all the changes in one update.
Thanks again,
Heather
 
From: "Stelling, Deanna L" <dstellin@fredhutch.org>
Date: Monday, February 7, 2022 at 9:26 AM
To: Jackie Dahlgren <jdahlgre@fredhutch.org>, "Kincaid, Heather L (US 398G-Affiliate)" <heather.kincaid@jpl.nasa.gov>
Cc: "Zhao, Yingqi" <yqzhao@fredhutch.org>
Subject: [EXTERNAL] RE: BRSI Imaging Set
 
Hi Heather,
Here are the IDs for the BRS-I training set. Any IDs not on this list should be moved to the Validation Set.
The exception would be Retrospective participants (IDs starting with “ER” at Moffit and IDs less than 2614 at Duke). They are not in either set.

Jackie, how do you want the retrospectives to be handled? Or do we need to ask Yingqi and the PIs?
 
Thank you,
Deanna
 
 
*******************
Deanna Stelling
CDE specialist, EDRN & COMPASS
Fred Hutchinson Cancer Research Center
1100 Fairview Ave. N., Mail Stop M3-A306
Seattle, WA 98109

WFH 206.446.0108
O 206.667.7388 (not in use)

Image removed by sender.
 fredhutch.org
 
From: Dahlgren, Jackie <jdahlgre@fredhutch.org> 
Sent: Monday, February 7, 2022 8:40 AM
To: Kincaid, Heather L (398G-Affiliate) <heather.kincaid@jpl.nasa.gov>
Cc: Stelling, Deanna L <dstellin@fredhutch.org>; Zhao, Yingqi <yqzhao@fredhutch.org>
Subject: BRSI Imaging Set
 
Hi Heather,
 
Deanna can send you the ID#s of the people in the Validation Set to-date and then please move them into their own collection that will always be private and only accessible to those approved to receive it.  This will need to be done again once all the Pending people are Confirmed.  Thanks - Jackie
 
Jackie Dahlgren (she/her)
